### PR TITLE
Add README content, simplify core model, fix json formatting of output and datetime parsing for hollywoodtheatre.org

### LINKED ISSUE #10: Update `README` with command line options
Documentation request:
Update `README` with info on command line argument usage

### PR DESCRIPTION
**Pull request by drewfead**
Fixes #1 
Fixes #10

### CODE REVIEW

File: internal/hollywood/datetime.go [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package hollywood
+
+import "time"
+
+func ParseDateTime(monthAndDay, clockTime string, timeZone *time.Location) (time.Time, bool) {
+	d, err := time.ParseInLocation("Monday, January 2", monthAndDay, timeZone)
+	if err != nil {
+		return time.Time{}, false
+	}
+	t, err := time.ParseInLocation("3:04 PM", clockTime, timeZone)
```
Comment (jamespullar) [inline internal/hollywood/datetime.go:10]: What's with the specific date and time values passed to `ParseInLocation`? Are these default values?

Comment (drewfead) [reply internal/hollywood/datetime.go:10]: Good question, these are from the canonical timestamp used to specify go date formats (one of the languages stranger conventions, to be sure). See more [here](https://cs.opensource.google/go/go/+/master:src/time/format.go;l=9-122)

Comment (jamespullar) [reply internal/hollywood/datetime.go:10]: 🧠 💥  That is quite a quirky convention. Could these be moved to const values somewhere?

**Review by jamespullar — APPROVED** [review]

**Comment by drewfead** [comment]
Broken out:
#13 #14 #15 #16 #17 #18 #19 

More changes to come after those are merged

**Pull request closed without merging**